tests/raster/test_imports.py

```diff
@@ -81,8 +81,10 @@
 @pytest.mark.parametrize(
     "convert_option",
```

Comment:
use a list of [param](https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-param) instances to pass multiple args into the test function, rather than trying to encode all the arguments in one string

eg the argument names here could be `extra_options` (list), `input` (str)

Reply:
Done